applications/data_analysis/humid_heat_metrics.ipynb

```diff
@@ -0,0 +1,392 @@
+{
```

Comment:
<p>Would it be helpful to include more details about "ERA5 renanalysis data"and the "Additional methods that may be considered and investigated in the future are: Hunter, Tonouchi, Liljegren, Li." are? For clarification for a new reader, either with a link to a webpage or a sentence summary</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>+1 here</p>

Reply:
I think I'll remove the "additional methods" and keep that as an internal future work note.